### PR TITLE
Typo and grammar fixes

### PR DESCRIPTION
**Pull request by Toasterbirb**
Various typo/grammar fixes. Some parts might be up for debate

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -34,11 +34,11 @@ export default function Index({ redirectOrigin }) {
       <Header>
         <Heading.H1>Welcome!</Heading.H1>
         <Text>
-          There is litle understanding about mental disabilities, which is
+          There is only a little understanding about mental disabilities, which is
```
Comment (vilhelmprytz) [inline pages/index.js:37]:
```suggestion
          Most people have very little knowledge about mental disabilities, which is
```

Could this be rephrased like this? Maybe it doesn't work in this context.

**Review by vilhelmprytz — CHANGES_REQUESTED** [review]
I just incorporated some of these changes in #26 without realizing this PR already existed!

**Review by alvarlagerlof — APPROVED** [review]
Looks good to me

**Review by vilhelmprytz — APPROVED** [review]

File: pages/index.js (hunk 1 above)
Comment (Toasterbirb) [reply pages/index.js]: Most likely. I was a bit confused about the sentence and didn't know what it should say

**Pull request merged**